packages/dds/tree/src/simple-tree/node-kinds/array/arrayNode.ts

```diff
@@ -848,15 +848,15 @@ type Insertable<T extends ImplicitAllowedTypes> = readonly (
 	| IterableTreeArrayContent<InsertableTreeNodeFromImplicitAllowedTypes<T>>
 )[];
 
-abstract class CustomArrayNodeBase<const T extends ImplicitAllowedTypes>
+abstract class CustomArrayNodeBase<const T extends ImplicitAnnotatedAllowedTypes>
```

Comment:
This change means the annotations don't have to be removed by the derived classes. Having the annotations available here could be useful for some future annotations, but also allows the simpleSchema getter to include the annotation making the annotations available in more places at runtime as well.